Dockerfile

```diff
@@ -15,9 +15,10 @@ RUN mv omnetpp-$VERSION omnetpp
 WORKDIR /root/omnetpp
 ENV PATH /root/omnetpp/bin:$PATH
 # remove unused files and build
-RUN ./configure WITH_OSG=no　&& \
+RUN ./configure WITH_OSG=no && \
+    make -j $(nproc) MODE=debug base && \
     make -j $(nproc) MODE=release base && \
-    rm -r doc out test samples misc config.log config.status
```

Comment:
`misc` has the utilities for debugging through gdb by OMNET++ IDE
https://github.com/omnetpp/omnetpp/tree/master/misc/gdb